.github/workflows/rti-docker.yml

```diff
@@ -0,0 +1,30 @@
+name: Push RTI Image to Docker Hub
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    name: Build and push RTI to Docker Hub
+    steps:
+      - name: Check out lingua-franca repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0
+      - name: Check Lingua Franca Version
+        id: check_version
+        run: |
+          export LF_VERSION=$(cat core/src/main/resources/org/lflang/StringsBundle.properties | sed 's/VERSION = \([0-9]*\.[0-9]*\.[0-9]*\)-SNAPSHOT/\1/')
```

Comment:
I think it might make sense to keep a suffix that indicates that it's a snapshot version and not a release version.

Reply:
Yes. I changed the version to `$(cat core/src/main/resources/org/lflang/StringsBundle.properties | sed -n 's/.*VERSION = \(.*\)/\1/p' | tr '[:upper:]' '[:lower:]')`